src/popups/InfoFPWPopup/InfoFPWPopup.js

```diff
@@ -0,0 +1,114 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Feature from 'ol/Feature';
+import { withTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+import { compose } from 'lodash/fp';
+import Button from 'react-spatial/components/Button';
+import { IoIosArrowRoundBack, IoIosArrowRoundForward } from 'react-icons/io';
+
+import './InfoFPWPopup.scss';
+
+const propTypes = {
+  feature: PropTypes.instanceOf(Feature).isRequired,
+  language: PropTypes.string.isRequired,
+  t: PropTypes.func.isRequired,
+};
+
+const defaultProps = {};
+
+class InfoFPWPopup extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      imageIndex: 0,
+    };
+  }
+
+  setImageIndex(index) {
+    this.setState({
+      imageIndex: index,
+    });
+  }
+
+  render() {
+    const { feature, language, t } = this.props;
+    const { imageIndex } = this.state;
+
+    let text;
+    let image;
+    let pagination;
+
+    const pureText = feature.get(`${language}_text`) || feature.get('de_text');
+    if (pureText) {
+      // eslint-disable-next-line react/no-danger
+      text = <div dangerouslySetInnerHTML={{ __html: pureText }} />;
+    }
+
+    const photoPath = feature.get('photo_path');
+    if (photoPath) {
+      const photoPaths = [JSON.parse(photoPath), JSON.parse(photoPath)].flat();
+
+      image = (
+        <img
+          src={`${process.env.REACT_APP_CARTARO_1_URL}/${photoPaths[imageIndex]}`}
+          draggable="false"
+          alt={t('Kein Bildtext')}
+        />
+      );
+
+      if (photoPaths.length > 1) {
+        pagination = (
```

Comment:
With the new baselayer and the new resolutions, it is possible that a popup opens with more than one feature. And if a feature has images (currently only on stag), we would see two pagination lines with `1 of 2`. As a solution, we could change the data. We could restrict the popup to only one feature. Or we could change the styling. What do you think?

Reply:
Yes using the same pagination is not so nice.

Ideally a simple slideshow should be nice for images: https://react-slideshow.herokuapp.com/.
The lighter you get the better it is.

Reply:
You can also create your own component, or just change the style of what you've already done (maybe the fatetr way).

Reply:
I have integrated the react-slideshow-image lib.